foursquare/tests/test_venues.py

```diff
@@ -67,6 +67,9 @@ def test_search_browse(self):
         response = self.api.venues.search(params={'ll': self.default_geo, 'radius': self.default_geo_radius, 'intent': 'browse'})
         assert 'venues' in response
 
+    def test_claim_venues(self):
+        response = self.api.venues.claim(self.default_venueid, params={"description": "Venue is Claimed"})
```

Comment:
Ah, this one won't actually pass. Sorry about that.
The test suite isn't very good at testing POSTs. In this case, I don't own the default venue ID, so I won't be able to claim it.
Additionally, "description" isn't a documented param to pass to the call.